test/e2e/common_suite_test.go

```diff
@@ -445,3 +445,38 @@ func doTestPodVMwithAnnotationsLargerCPU(t *testing.T, assert CloudAssert) {
 	}
 	newTestCase(t, "PodVMwithAnnotationsLargerCPU", assert, "Failed to Create PodVM with Annotations Larger CPU").withPod(pod).withInstanceTypes(testInstanceTypes).withCustomPodState(v1.PodPending).run()
 }
+
+func doTestExtraPods(t *testing.T, assert CloudAssert) {
```

Comment:
All the other test cases are have one test content related name, can we change this function name to 
`doTestServiceAndExtraPod`